server/api.go

```diff
@@ -489,9 +489,16 @@ func (p *Plugin) getUnreads(w http.ResponseWriter, r *http.Request, userID strin
 			continue
 		}
 
+		issueURL := n.GetSubject().GetURL()
+		issueNum := string(issueURL[len(issueURL)-1])
+		subjectUrl := n.GetSubject().GetURL()
```

Comment:
There is a convention to capitalize `URL` like so:
```suggestion
		subjectURL := n.GetSubject().GetURL()
```